File: day56/code.cpp
// Leetcode Solution

// class Solution {
// public:
//     int sum(int n)
//     {
//         int summ=0, ans;
//         while(n)
//         {
//             summ+=n%10;
//             n/=10;
//         }
//         if(summ>9)
//             ans=sum(summ);
        
//         else
//             ans=summ;
        
//         return ans;
//     }
//     int addDigits(int num) {
//         return sum(num);
//     }
// };

File: day69/code.py
#Leetcode Solution

# class Solution:
#     def myPow(self, x: float, n: int) -> float:
#         if n == 0:
#             return 1
        
#         if n < 0:
#             x**=-1
#             n*=-1
        
#         if n % 2 == 1:
#             return x * self.myPow (x,n-1)
#         else:
#             num = self.myPow(x,n//2)
#             return num * num